chainer/functions/normalization/l2_normalization.py

```diff
@@ -26,23 +26,21 @@ def forward(self, inputs):
         self.retain_inputs((0,))
         x, = inputs
         xp = cuda.get_array_module(x)
-        norm = xp.linalg.norm(x, axis=self.axis) + self.eps
-        norm = xp.expand_dims(norm, self.axis)
+        norm = xp.linalg.norm(x, axis=self.axis, keepdims=True) + self.eps
```

Comment:
According [the documentation](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.linalg.norm.html), `numpy.linalg.norm(keepdims=)` has been introduced in 1.10.0. We should avoid using it.